engine/src/main/java/com/datatorrent/stram/util/LoggerUtil.java

```diff
@@ -355,4 +361,66 @@ private static boolean isErrorLevelEnable()
     return false;
   }
 
+  /**
+   * Adds Logger Appender
+   * @param name Appender name
+   * @param className Appender class name
+   * @param args Appender args
+   */
+  public static void addAppender(String name, String className, Object args)
+  {
+    if (getAppendersNames().contains(name)) {
```

Comment:
Yes, the method getRootLogger().getAppender(name) can be used to check if an appender exists. But I think the method that returns names of all appenders should be in the basic list of logger util methods.